docs/basics/stable-versions.md

```diff
@@ -35,4 +35,28 @@ You can now access the stable Nextcloud instance at [http://stable28.local](http
 
 ## Apps without stable branches
 
-Some apps do not have stable branches or cover multiple Nextcloud version. You can use the `ADDITIONAL_APPS_PATH` variable in your `.env` file to add a cloned app to all Nextcloud containers. By default, this is set to `./data/apps-extra`
+Some apps do not have stable branches or cover multiple Nextcloud version. You can use the `ADDITIONAL_APPS_PATH` variable in your `.env` file to add a cloned app to all Nextcloud containers:
```

Comment:
Did you do any adjustments on the .env file for that? Maybe we can make this description even simpler by just telling people to clone such apps into `data/apps-extra` ?